<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibratioDialog</class>
 <widget class="QDialog" name="CalibratioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>801</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_Chessboard">
      <property name="title">
       <string>Vision-Based-Controller</string>
      </property>
      <widget class="QCheckBox" name="checkBox_vbc">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Vision-Based-Controller Calibration</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_no_trainingdata">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>5000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_vbc_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Amount of training images (default: 5000)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vbc_savedata">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Save taken images</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vbc_calibratecolor">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate Color Threshold beforehand</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_vbc_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Estimated time for capturing training data:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_vbc_time">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>220</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NAN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_vbc_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>471</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Please activate the custom TCP &quot;Chesster_training&quot; before starting the training procedure and remove the chessboard from the working space!</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vbc_skip">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Skip Data generation and use available data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_usedata">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>150</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Note: Place data in resources/VBC_Data/ and name it &quot;training_data_Input.csv&quot;, &quot;training_data_Output.csv&quot;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_Vision_Based_Control">
      <property name="title">
       <string>Detector (Chessboard)</string>
      </property>
      <widget class="QCheckBox" name="checkBox_detector">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Detector-Calibration</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_detector_show_debug">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Debug Images</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Start Calibration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
